app/javascript/lib/vue-components/modules/Table.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div
+    data-testid="table-component"
+    class="gobierto-table"
+  >
+    <div class="gobierto-table__header">
+      <slot name="title" />
+      <slot
+        name="columns"
+        :toggle-visibility="toggleVisibility"
+      />
+    </div>
+    <table>
+      <thead>
+        <template v-for="[id, { name, index, cssClass }] in columnsMapArr">
+          <th
+            :key="index"
+            class="gobierto-table__th"
+            :class="cssClass"
+            @click="handleTableHeaderClick(id)"
+          >
+            {{ name }}
+            <SortIcon
+              v-if="currentSortColumn === id"
+              :direction="getSorting(id)"
+            />
+          </th>
+        </template>
+      </thead>
+      <tbody>
+        <template
+          v-for="(item, index) in dataTable"
+        >
+          <router-link
+            :key="index"
+            :to="{ name: routingMember, params: {id: item[routingId] } }"
```

Comment:
What if I don't want to make the cells enroutable? What if `item` has no `routingId`? This template cannot be like that